sklearn/linear_model/logistic.py

```diff
@@ -610,7 +610,7 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
                                  "solver cannot handle multiclass with "
                                  "class_weight of type dict. Use the lbfgs, "
                                  "newton-cg or sag solvers or set "
-                                 "class_weight='auto'")
+                                 "class_weight='balanced'")
```

Comment:
cf https://github.com/scikit-learn/scikit-learn/pull/5008#discussion_r42640364

Indeed, this is not related to this PR